docs/getting-started-guides/README.md

```diff
@@ -20,13 +20,98 @@ certainly want the docs that go with that version.</h1>
+a kubernetes cluster from scratch. 
+
+### Local-machine Solutions
+Local-machine solutions create a single cluster with one or more kubernetes nodes on a single
+physical machine.  Setup is completely automated and doesn't require a cloud provider account.
+But their size and availability is limited to that of a single machine.
+
+The local-machine solutions are:
+  - [Local Docker-based](#local-solutions) (recommended starting point)
+  - [Vagrant](vagrant.md) (works on any platform with Vagrant: Linux, MacOS, or Windows.)
+  - [No-VM local cluster](locally.md) (Linux only)
+
+
+### Hosted Solutions
+[Google Container Engine](https://cloud.google.com/container-engine) offers managed Kubernetes
+clusters.
+
+### Turn-key Cloud Solutions
+These solutions allow you to create Kubernetes clusters on range of Cloud IaaS providers with only a
+few commands, and have active community support.
+- [GCE](gce.md)
+- [AWS](aws.md)
+- [Azure](coreos/azure/README.md)
+
+### Custom Solutions
+
+Kubernetes can run on a wide range of Cloud providers and bare-metal environments, and with many
+base operating systems.
+
+If you can find a guide below that matches your needs, use it.  It may be a little out of date, but
+it will be easier than starting from scratch.  If you do want to start from scratch, because you
+have special requirements, or just because you want to understand what is underneath a Kubernetes
+cluster, try the [Getting Started from Scratch](scratch.md) guide.
+
+If you are interested in supporting Kubernetes on a new platform, check out our [advice for
+writing a new solution](../../docs/devel/writing-a-getting-started-guide.md).
+
+#### Cloud
+These solutions are combinations of cloud provider and OS not covered by the above solutions.
+- [AWS + coreos](coreos.md)
+- [GCE + CoreOS](coreos.md)
+- [AWS + Ubuntu](juju.md)
+- [Joyent + Ubuntu](juju.md)
+- [Rackspace + CoreOS](rackspace.md)
+
+#### On-Premises VMs
+- [Vagrant](coreos.md) (uses CoreOS and flannel)
+- [CloudStack](cloudstack.md) (uses Ansible, CoreOS and flannel)
+- [Vmware](vsphere.md)  (uses Debian)
+- [juju.md](juju.md) (uses Juju, Ubuntu and flannel)
+- [Vmware](coreos.md)  (uses CoreOS and flannel)
+- [libvirt-coreos.md](libvirt-coreos.md)  (uses CoreOS)
+- [oVirt](ovirt.md)
+- [libvirt](fedora/flannel_multi_node_cluster.md) (uses Fedora and flannel)
+- [KVM](fedora/flannel_multi_node_cluster.md)  (uses Fedora and flannel)
+
+#### Bare Metal
+- [Offline](coreos/bare_metal_offline.md) (no internet required.  Uses CoreOS and Flannel)
+- [fedora/fedora_ansible_config.md](fedora/fedora_ansible_config.md)
+- [Fedora single node](fedora/fedora_manual_config.md) 
+- [Fedora multi node](fedora/flannel_multi_node_cluster.md) 
+- [Centos](centos/centos_manual_config.md)
+- [Ubuntu](ubuntu.md)
+- [Docker Multi Node](docker-multinode.md)
+
+#### Integrations
+- [Kubernetes on Mesos](mesos.md) (Uses GCE)
+
+## Table of Solutions
```

Comment:
Can we replace this with a link, move this "table of solutions" to its own document, and say something like:

"See all of the solutions in a table, ordered by last tested/supported product version."